portal/portal-impl/impl/pom.xml

```diff
@@ -62,6 +62,10 @@
                 <artifactId>entitybroker-utils</artifactId>
         </dependency>
         <dependency>
+                <groupId>org.sakaiproject.profile2</groupId>
+                <artifactId>profile2-api</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
A decent profile should be a core part of an LMS, IMHO. So, yes. "We are aiming to more tightly integrate profile activities into the portal with the aim of improving the Sakai collaborative experience" :)